src/base/Context.ts

```diff
@@ -435,6 +435,15 @@ export class Context {
     return identity;
   }
 
+  /**
+   * @hidden
+   *
+   * Retrieve the non-proxied Polkadot client
```

Comment:
(nit)

```suggestion
   *  Retrieve the polkadot.js promise client
```

Very nit, but if someone is getting the "regular" version they shouldn't have to worry about what a proxy version is.

I had to look at the code to figure out what a Proxy meant in this context. I first thought proxy server where the connection is going through some jump box or VPN.

Reply:
That makes sense. I have changed the docs as suggested.